data/src/main/java/com/linkedin/data/template/Custom.java

```diff
@@ -149,4 +151,16 @@ public static void initializeCustomClass(Class<?> customClass)
   {
     DataTemplateUtil.initializeClass(customClass);
   }
+
+  /**
+   * Return whether the input class is a Custom type or not
+   * uses coercer map to identify as each custom type must register a coercer excluding the primitive types.
+   *
+   * @param clazz is the class to check.
+   * @return true if an object of the input class is a Custom type.
+   */
+  public static boolean isCustomType(Class<?> clazz)
```

Comment:
I suggest renaming to `isCustomClass` to be consistent with the other methods in this class. Update the Javadoc to explain that this method checks if a given class is registered as a custom Java binding.

Reply:
removed the changes